README.md

```diff
@@ -1,5 +1,56 @@
 # CompassQL
-CompassQL Query Language for visualization recommendation.
+
+CompassQL is a visualization query language that powers chart recommendations in [Voyager 2](https://github.com/vega/voyager2).
+
+As described in our [vision paper](idl.cs.washington.edu/papers/compassql) and [Voyager 2](idl.cs.washington.edu/papers/voyager2) paper, a CompassQL query is a JSON object that contains the following components:
```

Comment:
Links that don't start with 'http(s)://' didn't work properly when I opened the file on Github.